src/store/components/AdminPreparePage/index.tsx

```diff
@@ -73,16 +75,26 @@ const AdminPreparePage: React.FC<AdminPreparePageProps> = (props) => {
           className="admin-prepare-page-table"
           size="small"
           rowKey="uuid"
-          dataSource={Object.values(merchData).map((data) => ({
-            uuid: data.name,
-            itemDisplay: (
-              <>
-                <h2>{data.name}</h2>
-                {data.variantType && data.variantValue && <h3>{`${data.variantType}: ${data.variantValue}`}</h3>}
-              </>
-            ),
-            quantity: data.quantity,
-          }))}
+          dataSource={Object.values(merchData)
+            .sort((a, b) => {
+              if (a.name < b.name) {
+                return -1;
+              }
+              if (a.name > b.name) {
+                return 1;
+              }
+              return 0;
+            })
+            .map((data) => ({
+              uuid: data.name,
+              itemDisplay: (
+                <>
+                  <h2>{data.name}</h2>
+                  {data.variantType && data.variantValue && <h3>{`${data.variantType}: ${data.variantValue}`}</h3>}
```

Comment:
more of a nitpick and similar to the function prop but they be more readable if extracted into a function or separate boolean like `doesVariantExist`

Reply:
I pulled `data.variantType && data.variantValue` into a separate variable.